### PR TITLE
feat(dataframe): support deleting items in DataFrame&ColumnSeries by loc

### PR DESCRIPTION
**Pull request by AChenQ**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2bpm5uq [parent] SDK 实现 Sheet 删除行的功能](https://app.clickup.com/t/2bpm5uq)

File: graviti/dataframe/column/indexing.py [hunk 1]
```diff
@@ -47,6 +47,12 @@ def __setitem__(
     ) -> None:
         self.obj.__setitem__(key, value)
 
+    def __delitem__(
+        self,
+        key: Union[int, slice],
```
Comment (linjiX) [inline graviti/dataframe/column/indexing.py:52]: remove the tailing comma?

Comment (linjiX) [inline graviti/dataframe/column/indexing.py:52]: `key` or `index`?

**Review by linjiX — CHANGES_REQUESTED** [review]

**Review by linjiX — APPROVED** [review]

File: graviti/dataframe/column/indexing.py [hunk 2]
```diff
@@ -47,6 +47,9 @@ def __setitem__(
     ) -> None:
         self.obj.__setitem__(key, value)
 
+    def __delitem__(self, key: Union[int, slice]) -> None:
```
Comment (linjiX) [inline graviti/dataframe/column/indexing.py:50]: support doing sth

**Review by linjiX — CHANGES_REQUESTED** [review]

File: graviti/dataframe/column/indexing.py (hunk 2 above)
Comment (linjiX) [reply graviti/dataframe/column/indexing.py:50]: in commit message, the "series" and "row" are mismatched?

**Review by linjiX — APPROVED** [review]

**Pull request merged**